test/functional/lib/tests.cpp

```diff
@@ -5062,6 +5062,8 @@ OffloadRssError()
     UINT16 IndirectionTableSize = 1 * sizeof(PROCESSOR_NUMBER);
     UINT32 RssConfigSize = sizeof(*RssConfig) + IndirectionTableSize;
 
+    TEST_EQUAL((HANDLE)3, GetCurrentThread());
```

Comment:
Can we have a comment on this? I don't quite understand what this means.

Reply:
good catch - i used this to make sure a test failed and the logs were emitted.